pages/en/blog/vulnerability/openssl-fixes-in-regular-releases-oct2023.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2023-10-25:00:15.000Z
+category: vulnerability
+title: OpenSSL Recent Security Patches
+slug: openssl-fixes-in-regular-releases-oct2023
+layout: blog-post.hbs
+author: Rafael Gonzaga
+---
+
+## Summary
+
+The vulnerabilities released in the OpenSSL Security Advisory of:
+
+- OpenSSL 3.0.10 - Tuesday 8th September 2023
```

Comment:
Something's gone awry here. OpenSSL 3.0.10 was released in August: https://mta.openssl.org/pipermail/openssl-announce/2023-August/000269.html
We've already updated to the quic fork of OpenSSL 3.0.10 in the August security updates: https://nodejs.org/en/blog/vulnerability/august-2023-security-releases